### PR TITLE
P-397 Add the new VC test case for ts-tests.

### PR DESCRIPTION
**Pull request by 0xverin**
### Context

resolves p-397

Now all vc basic tests(e.g. `di_vc`, `dr_vc`, `ii_vc` tests) get data from mock service, and real data-provider tests are not included in this test.
I have modified some rust code, please help to take a closer look, I hope it will not affect worker logic.

### CODE REVIEW

**Review by Traf333 — APPROVED** [review]
Overall looks good for me. Just one question came to my mind, what are the reasons to hardcode some paths within data providers like **"v1/blocks/e4068e6a326243468f35dcdc0c43f686/children"**. Is the `e406....f686` identifier permanent, or it might change in some cases?

File: tee-worker/litentry/core/assertion-build/src/oneblock/mod.rs [hunk 1]
```diff
@@ -38,8 +38,8 @@ pub struct OneBlockResponse {
 }
 
 impl RestPath<String> for OneBlockResponse {
-	fn get_path(path: String) -> core::result::Result<String, RestClientError> {
-		Ok(path)
+	fn get_path(_path: String) -> core::result::Result<String, RestClientError> {
+		Ok("v1/blocks/e4068e6a326243468f35dcdc0c43f686/children".to_string())
```
Comment (BillyWooo) [inline tee-worker/litentry/core/assertion-build/src/oneblock/mod.rs:42]: same question here. Why it is hard coded here.

**Review by BillyWooo — APPROVED** [review]
LGTM

**Comment by BillyWooo** [comment]
> Overall looks good for me. Just one question came to my mind, what are the reasons to hardcode some paths within data providers like **"v1/blocks/e4068e6a326243468f35dcdc0c43f686/children"**. Is the `e406....f686` identifier permanent, or it might change in some cases?

I think it's simply moving around the hard code place.

**Pull request merged**